README.md

```diff
@@ -1 +1,12 @@
 # hw01
+
+## My introduction
```

Comment:
Overall, it's a pretty concise but informational bio that has utilized RMarkdown well!